### DISCUSSION TITLE
manually edit the allowed hosts to add local ip address?

### DISCUSSION THREAD

**Discussion by openaspace** [modoboa/modoboa#2655] [Help]
I can manually edit the allowed hosts to add local ip address? I want to close the 443 ports on the wan.

ALLOWED_HOSTS = [
    'mail.server.digital',
]

**Comment by Spitfireap** [comment]
Hi,

If you want to do à reverse proxy, url will be passed in to your server, the host will correspond to mail.domain.tld, no need to change it :)

**Comment by openaspace** [comment]
No, I didn't make myself clear.

ports 80-443-993-587 are completely closed on the public network, only the 25 is opened.
I access the server ip on a private ip class with vpn, therefore I don't loose time with this fine configurations. 
Direct ip access and stop.

So.. my question is, I can manually modify the "allowed hosts" adding the vpn private ip or this modification will broke the way by modoboa write in the configuration file?

**Reply by openaspace** [reply]
So.. I can't manually modify the allowed hosts?

**Reply by Spitfireap** [reply]
Hey, you can modify the allowhost, there should be no issue. Restart uwsgi service or reboot afterward.

**Reply by tonioo** [reply]
you can, but unless you use a fqdn it won't work. With a regular setup, modoboa is behind a reverse proxy (nginx) which uses a virtualhost
